test/unit/storage/StorageConfig.test.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-underscore-dangle, object-curly-newline, object-curly-spacing */
```

Comment:
These two `object-curly-newline, object-curly-spacing` lead to inconsistence style in the project...

Reply:
Fixed